docs/source/guides/user_guide.rst

```diff
@@ -172,120 +172,25 @@ For example:
-
-Additionally, a log file for each job is located in the output folder, and any output grid files or images specified by the input configuration will be located in the respective job output folder.
-
-.. note:: You must specify the `out_dir` key in the input YAML configuation to use matrix expansion.
-
-Multiple dimensional matrix expansion is additionally supported. For example, the following configuation produces six jobs:
-
-.. code:: yaml
-
-    out_dir: 'out_dir'
-
-    matrix:
-      f_bedload:
-        - 0.5
-        - 0.4
-        - 0.2
-      h0:
-        - 1
-        - 5
-
-
-.. _ensemble_expansion_tag:
-
-Ensemble expansion
-------------------
-
-Ensemble expansion creates replicates of specified model configurations with different random seed values.
-Like the matrix expansion, the `out_dir` key must be specified in the input configuration file.
-The `ensemble` key can be added to any configuration file that does not explicitly define the random seed.
-As an example, two model runs can be generated with the same input sediment fraction using the following configuration `.yml`:
-
-.. code:: yaml
-
-    out_dir: 'out_dir'
-
-    f_bedload: 0.5
-    ensemble: 2
-
-This configuration file would produce two model runs that share the same parameters, but have different initial random seed values.
-The ensemble expansion can be applied to configuration files that include a matrix expansion as well:
-
-.. code:: yaml
-
-    out_dir: 'out_dir'
-
-    ensemble: 3
-
-    matrix:
-      h0:
-        - 1.0
-        - 2.0
-
-The above configuration file would produce 6 model runs, 3 with a basin depth (`h0`) of 1.0, and 3 with a basin depth of 2.0.
-
-.. _set_expansion_tag:
-
-Set expansion
--------------
-
-Set expansion enables user-configured parameter sets to take advantage of the :obj:`~pyDeltaRCM.Preprocessor` infrastructure (such as the job output preparation and ability to run jobs in parallel), while also enabling flexible configurations for parameter sets than cannot be configured via `matrix` expansion.
-For example, to vary `Qw0` while holding `Qs0` fixed requires modifying both `C0_percent` and some water-discharge-controlling parameter *simultaneously*; i.e., this cannot be achieved with `matrix` expansion. 
-
-To use set expansion, add the `set` key to a configuration file, and define a *list* of *dictionaries* which set the parameters of each run to be completed.
-For example, to configure two model runs, the first with parameters ``u0: 1.0`` and ``h0: 1.0``, and the second with parameters ``u0: 1.2`` and ``h0: 1.2``:
-
-.. code:: yaml
-
-    set:
-      - {u0: 1.0, h0: 1.0}
-      - {u0: 1.2., h0: 1.2}
-
-
-All jobs in the `set` specification must have the exact same set of keys.
-Moreover, additional `ensemble` or `matrix` specifications are not supported with the `set` specification. 
+Supporting documentation and files
+==================================
 
-.. include:: subsidenceguide.rst
+   /examples/slight_slope
```

Comment:
Is this meant to be a link to the advanced model configuration or the subsidence subclassing example?

Reply:
I guess it's supposed to just be a place to put useful references to other bits of documentation. Not an exhaustive list, necessarily. I've updated the list to include a few more items and fixed the links.